types/node/ts3.2/fs.d.ts

```diff
@@ -2,14 +2,14 @@
 /// <reference path="../fs.d.ts" />
 
 declare module 'fs' {
-    interface BigIntStats extends StatsBase<BigInt> {
+    interface BigIntStats extends StatsBase<bigint> {
```

Comment:
This also needs to be done in `types/node/v12/ts3.2/fs.d.ts`.

Reply:
hec